lib/vagrant-libvirt/config.rb

```diff
@@ -279,26 +279,26 @@ def _get_cdrom_dev(cdroms)
       end
 
```

Comment:
This file should probably also contain a block added to the 'merge' method to handle numa_nodes being defined in different files & blocks.